lfs.c

```diff
@@ -411,6 +411,17 @@ static inline void lfs_superblock_tole32(lfs_superblock_t *superblock) {
     superblock->attr_max    = lfs_tole32(superblock->attr_max);
 }
 
+static inline bool is_node_not_in_list(struct lfs_mlist *head,
+    struct lfs_mlist *node) {
```

Comment:
style nit: Two indents when inside parens (I realize I don't have this documented anywhere...)

Reply:
done